.github/workflows/validate-code.yml

```diff
@@ -63,7 +63,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        php: [ "7.4", "8.0", "8.1", "8.2" ]
+        php: [ "7.4", "8.0", "8.1", "8.2", "8.3", "8.4" ]
```

Comment:
@icanhazstring If you are interested, I can also swap the fixed array with a dynamic one using one of my actions: https://github.com/WyriHaximus/github-action-composer-php-versions-in-range?tab=readme-ov-file#example